### PR TITLE
Update site for Winter 2023 term

### PR DESCRIPTION
**Pull request by bandrson**
### Proposed Changes
- Update current leadership page
- Update past executives page
- Update current term on landing page
- Update copyright year in footer
- Fix assets and links that aren't using relative paths properly
- Add January 2023 newsletter

### CODE REVIEW

**Review by milolucero — APPROVED** [review]
I have reviewed the changes for updating the current and past board, and it all looks good to me!

**Review by milolucero — APPROVED** [review]
I have reviewed the changes for updating the current and past board, and it all looks good to me!

File: index.html [hunk 1]
```diff
@@ -33,7 +33,12 @@ <h3 style="text-align: center;">Bits and Bytes Association <br>Winter 2023</h3>
     box-shadow: 0px 0px 19px 8px rgba(0,0,0,0.4);">
         <div class="row" style="padding: 20px;">
             <h3 style="text-align: center;  color:white;">Read our latest newspaper posts!</h3>
-            <div class="col-md-2 col-md-offset-3 post-news" >
+            <div class="col-md-2 col-md-offset-3 post-news">
+                <a href="assets/newspapers/bba-newsletter-jan-2023-issue.pdf">
```
Comment (milolucero) [inline index.html:37]: Just for consistency, can we also add here the "Latest news paper link goes here" comment?

**Review by milolucero — CHANGES_REQUESTED** [review]
Please check the comment and apply the change for consistency.

File: index.html (hunk 1 above)
Comment (bandrson) [reply index.html:37]: Yep, I'll make the change.

**Comment by bandrson** [comment]
Requested change has been made.

**Review by milolucero — APPROVED** [review]
Perfect, thanks for the change!

**Comment by bandrson** [comment]
Squashed the typo fix onto the newsletter commit. Ready for merge now.

**Pull request merged**